oak/proto/grpc_encap.proto

```diff
@@ -28,15 +28,13 @@ import "third_party/google/rpc/status.proto";
 // simplified envelope format that includes the relevant details.
 
 message GrpcRequest {
```

Comment:
This would look even nicer if we were using FIDL or something like that, we could add the response channel handle to the request message.